### ISSUE TITLE
Use antd for ui

### ISSUE THREAD

**Issue by yasintz** [pass-wall/passwall-extension#1]
Instead of uikit, antd used on the web and desktop should be used.

**Comment by yakuter** [comment]
I agree. We must use almost same technologies in all clients. @bufgix, beacuse you are the author of this repo, asking you first. Do you want to work on this?

**Comment by bufgix** [comment]
My purpose is to keep the plugin size minimum. That's why I use UIKit. But if you want to change, we can switch to antd

**Comment by yakuter** [comment]
Is antd so big in size? I heard that is is fast.

**Comment by bufgix** [comment]
![resim](https://user-images.githubusercontent.com/22038798/80306252-be0eb580-87ca-11ea-8ce5-b94f1b487c3b.png)

Also, we use vue bundle of antd size may be bigger

**Comment by mtoygar** [comment]
Bundle size may not be an issue for an extension. I couldn't find a document reference for this but as far as I understand extensions store its resources locally. On my Mac extension sources stored in the below directory.
`HOME_DIRECTORY/Library/Application Support/Google/Chrome/Default/Extensions/EXTENSION_ID`

Since there is no network request to download resources, bundle size might be neglected. 

a similar opinion from stackoverflow.
https://stackoverflow.com/a/12963902
